docs/xchain.html

```diff
@@ -1,139 +1,236 @@
+            <input
+              type="text"
+              id="server"
+              placeholder="Enter Your Node Server"
+            />
             <label class="ui label" id="networkId"></label>
           </div>
         </div>
         <div class="field">
           <label>Request Key</label>
-          <input type="text" id="pact-id" placeholder="Enter Request Key">
+          <input type="text" id="pact-id" placeholder="Enter Request Key" />
         </div>
         <div id="pact-message" class="ui compact message hidden">
           <div class="header" id="pact-header"></div>
           <ul id="pact-info">
             <p>Source Chain: <span id="source-chain-id" /></p>
             <p>Target Chain: <span id="target-chain-id" /></p>
-            <p>Sender: <span id="sender" /></span>
-            <p>Receiver: <span id="receiver" /></span>
-            <p>Receiver Guard: <span id="rg" /></span>
-            <p>Amount: <span id="amount" /></span>
+            <p>Sender: <span id="sender" /></p>
+            <p>Receiver: <span id="receiver" /></p>
+            <p>Receiver Guard: <span id="rg" /></p>
+            <p>Amount: <span id="amount" /></p>
           </ul>
         </div>
-        <div id="submit-block">
-          <button id="submit-button" class="ui primary button" disabled>Finish Cross Chain Transfer</button>
+
+        <div id="gas-payer-form" class="ui form">
+          <div class="field">
+            <label>Gas payer account</label>
+            <input
+              type="text"
+              id="gas-payer"
+              placeholder="kadena-xchain-gas"
+              value="kadena-xchain-gas"
+            />
+            <!-- <button id="verify-gas-payer" class="ui primary button">Verify</button> -->
+          </div>
+
+          <div class="field">
+            <label>Gas price</label>
+            <input
+              type="text"
+              id="gas-price"
+              placeholder="0.00000001"
+              value="0.00000001"
+            />
+          </div>
+
+          <div class="field">
+            <label>Gas limit</label>
+            <input type="text" id="gas-limit" placeholder="750" value="750" />
+          </div>
+
+          <div id="sig-data-message" class="ui compact message hidden">
+            <h3 class="header">SigData</h3>
+            <span>Copy the output into a SigBuilder (e.g. Chainweaver)</span>
+            <textarea
+              name="SigData"
+              id="sig-data"
+              cols="30"
+              rows="10"
+              value=""
+              disabled="disabled"
+            ></textarea>
+            <div class="field"></div>
+            <button id="click-to-copy" class="ui secondary basic button">
+              Click To Copy
+            </button>
+          </div>
+
+          <div id="signed-transaction-wrapper" class="ui field hidden">
+            <label class="">Signed Transaction</label>
```

Comment:
Maybe `Signed Transaction (JSON)` instead to let the user know that they are using the `JSON` tab of sigbuilder results instead of the `CommandJSON` tab

Reply:
☑️